src/mnelab/dialogs/append.py

```diff
@@ -90,16 +90,23 @@ def dropEvent(self, event):
                     item.setTextAlignment(
                         Qt.AlignLeft | Qt.AlignVCenter
                         if col == 1
-                        else Qt.AlignRight | Qt.AlignVCenter
+                        else Qt.AlignVCenter | Qt.AlignVCenter
```

Comment:
Typo?

Reply:
Yes, remnants of the styling iterations. Just corrected